sass/directives/02_base_components/boxes/_boxes.scss

```diff
@@ -90,6 +90,21 @@
 
 // Item Bar
 
+$_item-bar-breakpoint: 750px;
```

Comment:
For the team's edification - we should come up with a rule for when to use `_` as a prefix to signify something special in our Sass. What's your rule for deciding whether or not to use this?